Added Project Youtube Clone

mrkc2303 opened the pull request:
# Description

A project which aims to clone YouTube UI with responsiveness by using HTML & CSS.

Fixes #18 
<!---give the issue number you fixed----->

## Type of change

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

- [x] New Project / Docs
 
# Checklist:
<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK 

![image](https://user-images.githubusercontent.com/67221487/128750048-855ac53c-1483-4347-adae-c25e56117093.png)
<br>
![image](https://user-images.githubusercontent.com/67221487/128750140-262c1d2a-2769-46d3-9b0c-0b96b4973dce.png)
<br>
![image](https://user-images.githubusercontent.com/67221487/128750244-d0902e09-07e3-4af9-bd45-7e7ff246b11a.png)

https://mrkc2303.github.io/youtube-clone/

Sanskriti2401 commented:
@mrkc2303 Deploy your project in github pages and share the link.
Move all the images inside assets->media

mrkc2303 commented:
@Sanskriti2401 
I have updated the files. I have also added the link to deployed project.
https://mrkc2303.github.io/youtube-clone/

Note: Working of this link and actual project my look different as i am constantly updating the project for enhancing my skills.

Kindly check and update ASAP

Sanskriti2401 reviewed (approved):
Great Work :)
